ivadomed/scripts/download_data.py

```diff
@@ -0,0 +1,235 @@
+    with os.scandir(extraction_folder) as it:
+        has_dir = False
+        nb_entries = 0
+        for entry in it:
+            if entry.name in ("__MACOSX",):
+                continue
+            nb_entries += 1
+            if entry.is_dir():
+                has_dir = True
+
+    if nb_entries == 1 and has_dir:
+        # tarball with single-directory -> go under
+        with os.scandir(extraction_folder) as it:
+            for entry in it:
+                if entry.name in ("__MACOSX",):
+                    continue
+                bundle_folder = entry.path
+    else:
+        # bomb scenario -> stay here
+        bundle_folder = extraction_folder
+
+    # Copy over
+    for cwd, ds, fs in os.walk(bundle_folder):
+        ds.sort()
+        fs.sort()
+        ds[:] = [d for d in ds if d not in ("__MACOSX",)]
+        for d in ds:
+            srcpath = os.path.join(cwd, d)
+            relpath = os.path.relpath(srcpath, bundle_folder)
+            dstpath = os.path.join(dest_folder, relpath)
+            if os.path.exists(dstpath):
+                # lazy -- we assume existing is a directory, otherwise it will crash safely
+                logger.debug("- d- %s", relpath)
+            else:
+                logger.debug("- d+ %s", relpath)
+                os.makedirs(dstpath)
+
+        for f in fs:
+            srcpath = os.path.join(cwd, f)
+            relpath = os.path.relpath(srcpath, bundle_folder)
+            dstpath = os.path.join(dest_folder, relpath)
+            if os.path.exists(dstpath):
+                logger.debug("- f! %s", relpath)
+                logger.warning("Updating existing “%s”", dstpath)
+                os.unlink(dstpath)
+            else:
+                logger.debug("- f+ %s", relpath)
+            shutil.copy(srcpath, dstpath)
+
+    logger.info("Removing temporary folders...")
+    shutil.rmtree(os.path.dirname(tmp_file))
+    shutil.rmtree(extraction_folder)
+
+
+def main(args=None):
+    # Dictionary containing list of URLs for data names.
+    # Mirror servers are listed in order of decreasing priority.
+    # If exists, favour release artifact straight from github
+
+    dict_url = {
+        "data_example_spinegeneric": ["https://github.com/ivadomed/data_example_spinegeneric/archive/master.zip"],
+        "data_microscopy_SEM_example": ["https://github.com/ivadomed/data_microscopy_SEM_example/archive/master.zip"],
+        "data-testing": ["https://github.com/ivadomed/data-testing/archive/master.zip"],
+        "t2_tumor": ["https://github.com/ivadomed/t2_tumor/archive/master.zip"],
+        "t2star_sc": ["https://github.com/ivadomed/t2star_sc/archive/master.zip"],
+        "mice_uqueensland_gm": ["https://github.com/ivadomed/mice_uqueensland_gm/archive/master.zip"],
+        "mice_uqueensland_sc": ["https://github.com/ivadomed/mice_uqueensland_sc/archive/master.zip"],
+        "findcord_tumor": ["https://github.com/ivadomed/findcord_tumor/archive/master.zip"]
```

Comment:
no, we should use the immutable artifacts generated by [distriploy](https://github.com/neuropoly/distriploy). E.g.: https://github.com/ivadomed/t2_tumor/releases/download/r20200621/r20200621_t2_tumor.zip

Reply:
because of https://github.com/neuropoly/distriploy/issues/7 we can go ahead with a simple tag

Reply:
Started in 3fafe8f. Winting on release from data-testing to finish up.